images/linux/scripts/installers/go.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+################################################################################
+##  File:  go.sh
+##  Desc:  Installs go, configures GOROOT, and adds go to the path
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+golangTags="/tmp/golang_tags.json"
```

Comment:
nitpick: enter between 9 and 10 lines for readability

Reply:
Done.